cypress/e2e/admin/adminRegistersStudentWithLink.feature

```diff
@@ -1,38 +1,75 @@
-        And Going to student registration page
-        And I am registering as student with email 'john.student@mail.tld' on '<env>'
-        Then I am able to logout
+        # Admin creates a new student
+        When I click on administration in menu
+        When I go to '<role_to_manage>' administration
+        When I click on FAB
+        When I click on Add User in opened FAB for '<role_to_manage>'
+        When I fill out the user creation form for '<user_firstname>' '<user_lastname>' with email '<user_email>'
+        When I click on add button to add '<role_to_manage>'
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        Then I can see the user with email '<user_email>' in the table
 
-        Examples:
-            | env | user       |
-            | dbc | admin1_dbc |
+        # Admin generates the registeration link to proceed with student registration
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        When I click edit '<role_to_manage>' button for '<user_email>'
+        Then I click on the button Generate Personal Registration Link for student
+
+        # Student performs registration via the generated link with PIN
+        Then I visit to the generated student registration link
+        When I choose the language for the registration process
+        Then I click on the button Next to proceed to the age selection
+        When I select the age over 16 years for registration
+        Then I click on the button Next to proceed to the personal data information page
+        Then I see my first name '<user_firstname>'
+        Then I see my last name '<user_lastname>'
+        When I click on the button Next to proceed to the next step
+        When I accept the privacy and terms of use consents
+        Then I click on the button Next to proceed to the registration pin step
+        When I request a new registration pin
+        Then I retrieve the registration pin to enter it into the form for '<namespace>'
+        Then I click on the button Send and Get Started to successfully complete the registration process
+        Then I see the page user data summary
+
+        # Newly registered student does the first login
+        When I visit the url for first login
+        When I enter the email assigned during user creation
+        When I enter the initial generated password
+        When I click on the button Login
+        Then I see the first login page section 1
+        When I click on the button Next in section 1
+        Then I see the section 2
+        When I click on the button Next in section 2
+        Then I see the section 3
+        Then I set a new password
+        Then I re enter the new password
+        When I click on the button Next in section 3
+        Then I click on the button Get started now in section 4
+        Then I see the dashboard
+        Then I logout from the application
```

Comment:
we don't need that step

Reply:
I this tests its needed, also due to session, can you try in your local gui cy?

Reply:
done, removed from this features.